examples/platforms/cc1352/radio.c

```diff
@@ -1896,11 +1896,16 @@ static void cc1352RadioProcessReceiveQueue(otInstance *aInstance)
 
             if (crcCorr->status.bCrcErr == 0 && (len - 2) < OT_RADIO_FRAME_MAX_SIZE)
             {
-#if OPENTHREAD_ENABLE_RAW_LINK_API
                 // TODO: Propagate CM0 timestamp
-                receiveFrame.mInfo.mRxInfo.mMsec = otPlatAlarmMilliGetNow();
-                receiveFrame.mInfo.mRxInfo.mUsec = 0; // Don't support microsecond timer for now.
+#if OPENTHREAD_CONFIG_ENABLE_TIME_SYNC
```

Comment:
These platforms don't support time sync feature yet, is it necessary to add the check here?

Reply:
I add this to prevent invalid usage.

Reply:
This seems to make the check in #4008 redundant. Is there value in having both? I believe the check in #4008 will prevent reaching this more useful error message.

Reply:
Is it better to have the check as in #4008 or the check that was proposed here?

Reply:
I think it's better to check here, because it give more useful information. I will remove the check introduced in #4008.